lib/Cake/Utility/CakeTime.php

```diff
@@ -1075,12 +1075,19 @@ public static function i18nFormat($date, $format = null, $default = false, $time
  * 	Or one of DateTimeZone class constants (PHP 5.3 and above)
  * @param string $country A two-letter ISO 3166-1 compatible country code.
  * 	This option is only used when $filter is set to DateTimeZone::PER_COUNTRY (available only in PHP 5.3 and above)
- * @param bool $group If true (default value) groups the identifiers list by primary region
+ * @param bool|array $group If true (default value) groups the identifiers list by primary region.
+ * 	Otherwise, an array containing `group` and `abbr` keys, to indicate that results are grouped and contain the
+ * 	timezone abbreviation.
  * @return array List of timezone identifiers
  * @since 2.2
  * @link http://book.cakephp.org/2.0/en/core-libraries/helpers/time.html#TimeHelper::listTimezones
  */
-	public static function listTimezones($filter = null, $country = null, $group = true) {
+	public static function listTimezones($filter = null, $country = null, $options = array()) {
+		$options = array_merge(array(
+			'group' => true,
+			'abbr' => false,
+		), $options);
```

Comment:
This isn't fully backwards compatible as sometimes $options will be a boolean. Which cannot be array merged.